src/network/getters/getSponsorshipDailyBuckets.ts

```diff
@@ -0,0 +1,38 @@
+import getGraphClient from '~/getters/getGraphClient'
+import {
+    GetSponsorshipDailyBucketsDocument,
+    GetSponsorshipDailyBucketsQuery,
+    GetSponsorshipDailyBucketsQueryVariables,
+    SponsorshipDailyBucket_Filter,
+} from '~/generated/gql/network'
+
+export const getSponsorshipDailyBuckets = async (
+    sponsorshipId: string,
+    dateLowerThan: string, // utc timestamp in seconds
```

Comment:
What about `fromTimestamp` and `toTimestamp`, both numbers?

Reply:
dateLowerThan/dateGreaterEqualThan are more explicit - you know exactly what it is, while with fromTimestamp/toStimstamp you might not be sure about the direction - is "from" the lower value or the higher value. I'd leave it as it is - eventually change it to number